test/positivity.lean

```diff
@@ -273,8 +273,11 @@ example : 0 ≤ Real.log 1 := by positivity
 example : 0 ≤ Real.log 0 := by positivity
 example : 0 ≤ Real.log (-1) := by positivity
 
-example {V : Type _} [NormedCommGroup V] (x : V) : 0 ≤ ‖x‖ := by positivity
-example {V : Type _} [NormedAddCommGroup V] (x : V) : 0 ≤ ‖x‖ := by positivity
+example {V : Type*} [NormedCommGroup V] (x : V) : 0 ≤ ‖x‖ := by positivity
+example {V : Type*} [NormedCommGroup V] {x : V} (hx : x ≠ 0) : 0 < ‖x‖ := by positivity
```

Comment:
btw, this comes from a different test that is failing:

```lean
import Mathlib.Tactic.Positivity.Basic

set_option trace.Tactic.positivity true

example {V : Type*} [Zero V] [PartialOrder V] {x : V} (hx : 0 < x) : x ≠ 0 := by positivity --fails???
example {V : Type*} [Zero V] [PartialOrder V] {x : V} (hx : 0 < x) : x ≠ 0 := ne_of_gt hx
```

I'm not sure why this fails, I'm currently trying to debug but it's annoying to debug tactics.

Reply:
I fixed this in #10293.